modules/misc/news.nix

```diff
@@ -108,6 +108,14 @@ in
   config = {
     news.entries = [
       {
+        time = "2017-09-10T00:15:19+02:00";
+        message = ''
```

Comment:
Looking good but should have a field `condition = config.programs.zsh.enable;` to limit the news to those that are using Zsh.